### PR TITLE
Add travel, eCommerce, and social sites to the mix

### PR DESCRIPTION
**Pull request by Fuzion24**
I really like this idea of giving all of the major sites score cards on TLS.  It may be nice to add customized notes to each site about what could potentially be leaked or what is given up.

Here is a related Tumblr.  http://httpshaming.tumblr.com/

Here is a SS of some of the sites I added:

screen shot 2015-01-18 at 8 45 16 pm

### CODE REVIEW

File: sites.json [hunk 1]
```diff
@@ -31,6 +31,120 @@
...
+                {
+                    "name": "Walmart",
+                    "domain": "www.walmart.com"
+                },
+                {
+                    "name": "CVS",
+                    "domain": "www.cvs.com"
+                },
+                {
+                    "name": "HomeDepo",
```
Comment (alex) [inline sites.json]: Dropped a "t" here

Comment (Fuzion24) [inline sites.json]: Fixed.

File: sites.json [hunk 2]
```diff
@@ -31,6 +31,120 @@
...
+                "name": "LinkedIn",
+                "domain": "www.linkedin.com"
+              },
+              {
+                "name": "Facebook",
+                "domain": "www.facebook.com"
+              },
+              {
+                "name": "Twitter",
+                "domain": "www.twitter.com"
```
Comment (gutworth) [inline sites.json]: Probably this should be just ``twitter.com``, since they redirect ``www``.

**Comment by alex** [comment]
There's some merge conflicts here now.

File: sites.json [hunk 3]
```diff
@@ -31,6 +31,120 @@
...
+                "domain": "www.match.com"
+              },
+              {
+                "name": "OkCupid",
+                "domain": "www.okcupid.com"
+              }
+           ]
+        },
+        {
+            "name": "E-Commerce Sites",
```
Comment (jayschwa) [inline sites.json]: I suggest renaming to "Commerce". We know they're sites and I think the web is mature enough that we can start dropping the `e-` on things :wink:

File: sites.json [hunk 4]
```diff
@@ -31,6 +31,120 @@
...
+                {
+                  "name": "Apple",
+                  "domain": "store.apple.com"
+                },
+                {
+                    "name": "Amazon",
+                    "domain": "amazon.com"
+                },
+                {
+                    "name": "Ebay",
```
Comment (jayschwa) [inline sites.json]: Nit: the correct case is "eBay".

**Comment by gutworth** [comment]
Merged manually in 60fab2f14c6c1bf9727f22390561e7ead8fbdde9 and addressed @jayschwa's nits.

**Pull request merged**